tests/e2e/cypress/integration/Resources-status/01-listing.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Filter a list of Resources
+  As a user
+  I want to apply filter(s) on a list of Resources
+  So that I can quickly view a specific group of these Resources
+
+  Background:
+    Given There are available resources
+
+  Scenario: I first access to the page
+    Then the unhandled problems are displayed
```

Comment:
Missing `When`
Could be for example : 
`When I filter on unhandled problems`
`Then Only non-ok resources are displayed`